core/store/orm/orm.go

```diff
@@ -1303,6 +1304,30 @@ func (orm *ORM) FindEncryptedP2PKeys() (keys []p2pkey.EncryptedP2PKey, err error
 	return keys, orm.DB.Find(&keys).Error
 }
 
+// CreateEncryptedOCRKeys creates an encrypted OCR private key record
+func (orm *ORM) CreateEncryptedOCRKeys(keys *ocrkey.EncryptedOCRPrivateKey) error {
+	return orm.DB.Create(keys).Error
+}
+
+// FindEncryptedOCRKeys finds all the encrypted OCR key records
+func (orm *ORM) FindEncryptedOCRKeys() (keys []*ocrkey.EncryptedOCRPrivateKey, err error) {
+	var result []ocrkey.EncryptedOCRPrivateKey
+	err = orm.DB.Find(&result).Error
+	if err != nil {
+		return keys, err
+	}
+	for i := 0; i < len(result); i++ {
```

Comment:
redundant you can scan straight into the slice

Why return a slice with pointers anyway?